### PR TITLE
Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-ansible-role-with-test-user.git`](https://github.com/cisagov/skeleton-ansible-role-with-test-user.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ local machine.
 Before the build user can be created, the following profile must exist in
 your AWS credentials file:
 
-* `cool-terraform-backend`
+- `cool-terraform-backend`
```
Comment (mcdonnnj) [inline README.md:22]: This role's Terraform is setup to run against the CyHy account and not the COOL multi-account configuration. I believe this profile should be changed to something more appropriate as a result.

**Review by mcdonnnj — CHANGES_REQUESTED** [review]
Just that AWS credentials profile again.

File: README.md (hunk 1 above)
Comment (jsf9k) [reply README.md:22]: See cisagov/skeleton-ansible-role-with-test-user#22.

**Review by dav3r — APPROVED** [review]
👍

**Review by mcdonnnj — APPROVED** [review]

**Pull request merged**